console/console-init/ui/src/modules/address/components/ExpiryQueue/ExpiryQueue.tsx

```diff
@@ -20,31 +20,31 @@ const styles = StyleSheet.create({
     flex: "1"
   }
 });
-export interface IExpiryQueueProps {
-  onExpiryQueueSelect?: (value: string) => void;
-  expiryQueue: string;
+export interface IExpiryAddressProps {
+  onExpiryAddressSelect?: (value: string) => void;
+  expiryAddress: string;
   deadletterOptions?: IDropdownOption[];
 }
-export const ExpiryQueue: React.FunctionComponent<IExpiryQueueProps> = ({
-  expiryQueue,
-  onExpiryQueueSelect,
+export const ExpiryAddress: React.FunctionComponent<IExpiryAddressProps> = ({
```

Comment:
Change folder and file name with postfix Address
`ExpiryAddress/ExpiryAddress.tsx`